### PR TITLE
Create test_concept

### PR DESCRIPTION
**Pull request by SansoneG**

### CODE REVIEW

File: doc/test_concept.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
+# PSENscan test concept
+
+![test architecture](test_architecture.svg)
+
+## ROS-dependent code
+
+### ROS parameter handler
```
Comment (ct2034) [inline doc/test_concept.md]: we need a section about acceptance tests

**Review by ct2034 — DISMISSED** [review]

File: doc/test_concept.md (hunk 1 above)
Comment (SansoneG) [reply doc/test_concept.md]: Done

**Review by jschleicher — COMMENTED** [review]
Great! That looks really helpful.
Maybe add a link here from the main Readme.md developers section (branching model paragraph?) or include it from doxygen?

**Comment by jschleicher** [comment]
@SansoneG Should this be merged? Or closed?

**Comment by SansoneG** [comment]
> @SansoneG Should this be merged? Or closed?

Merged. Needs approval though.

**Review by cdoehn — APPROVED** [review]

**Pull request merged**